src/format.ts

```diff
@@ -13,20 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import chalk from 'chalk';
+import * as jsdiff from 'diff';
+import * as entities from 'entities';
 import * as fs from 'fs';
 import * as path from 'path';
+
 import {Options} from './cli';
 import {createProgram} from './lint';
-import { promisify } from 'util';
-import { start } from 'repl';
+import {readFilep} from './util';
+
+interface replacement {
```

Comment:
Interfaces names should be capitalized. Also, could you add a JSDoc describing the role of this object? (By JSDoc I mean `/** */` style rather than `//` style)

Reply:
done